x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -158,13 +159,31 @@ async function resetCredentials(
         },
       }
     );
-    return {
-      password: response.data.password,
-      username: response.data.username,
-    };
-  } catch (error) {
-    throw new Error(`${error.message}`);
-  }
+    if (response.status !== 200) {
```

Comment:
Correct me if I'm wrong but `axios` throws an exception when the status isn't `2XX`. That means this condition will never be passed.